Makefile

```diff
@@ -97,6 +97,12 @@ test-e2e:
 	  KUBERNETES_CONFIG="$(KUBECONFIG)" $(GO) test -v -timeout 40m ./test/e2e/$$d -ginkgo.v -ginkgo.noColor -ginkgo.failFast || exit; \
 	done
 
+.PHONY: test-e2e-local
+test-e2e-local:
```

Comment:
Should we make the make target more specific as in https://github.com/openshift/cluster-node-tuning-operator/blob/131604110a23144fa0ccafc7be5a204a4cb6e4b5/Makefile#L202 ?

Reply:
I was wondering if we can have more tests that are e2e in nature, but do not require a running cluster.. but the name is not important obviously.

Reply:
if we are  planing to add a render lane in openshift/release then it should correspond to the make target

Reply:
@MarSik  we currently have https://github.com/openshift/cluster-node-tuning-operator/blob/master/hack/run-render-command-functests.sh under this repo.  If this target comes to replace the invocation of this script [like we did in PAO ](https://github.com/openshift-kni/performance-addon-operators/blob/1df302f67a2d8338af090a5084ccaef6e902b283/Makefile#L280) then we can remove the script (although the script contains some additional decorations)